packages/waas/src/auth.ts

```diff
@@ -363,6 +363,8 @@ export class SequenceWaaS {
   private async updateSessionStatus() {
```

Comment:
what calls the `updateSessionStatus()` method?

Reply:
isSIgnedIn calls it